/**:
  ros__parameters:
    airlight: 100.0     # 该数值越大，复原后的图像辉度越低，但除掉的噪音也越多。
    beta_b: 0.5         # 蓝光在水中的衰减程度，之后会在论文中说明。
    beta_g: 1.0         # 绿光在水中的衰减程度，之后会在论文中说明。
    beta_r: 2.0         # 红光在水中的衰减程度，之后会在论文中说明。
    is_show_fps: true   # 是否在处理后的图像上显示帧数。
    is_resize: false    # 如果直接处理360p的图像，应该不需要做resize。这里请进行实际测试。
    resize_scale: 0.5   # 默认意味着缩小原尺寸的一半。